### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sallemiines**
Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Update README with a How to test section for QA
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | Check README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ Check in one glance what is happening on your store with a list of KPI on your d
 
 PrestaShop: `1.7.7.0` or later
 
+## How to test
+Link to specs : https://docs.prestashop-project.org/functional-documentation/functional-documentation/ux-ui/back-office/improve/modules/product-comments-productcomments
+
+Link to test scenario : https://build.prestashop-project.org/test-scenarios/scenarios/modules/dashactivity.html
```
Comment (Hlavtox) [inline README.md:14]:
```suggestion
Link to test scenario: https://build.prestashop-project.org/test-scenarios/scenarios/modules/dashactivity.html
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ Check in one glance what is happening on your store with a list of KPI on your d
...
+
+In BO > Modules > Check that the module dashactivity is enabled
```
Comment (Hlavtox) [inline README.md:16]:
```suggestion
In `BO > Modules`, check that the module dashactivity is enabled.
```

File: README.md [hunk 3]
```diff
@@ -8,6 +8,19 @@ Check in one glance what is happening on your store with a list of KPI on your d
...
 PrestaShop: `1.7.7.0` or later
 
+## How to test
+Link to specs : https://docs.prestashop-project.org/functional-documentation/functional-documentation/ux-ui/back-office/improve/modules/product-comments-productcomments
+
+Link to test scenario : https://build.prestashop-project.org/test-scenarios/scenarios/modules/dashactivity.html
+
+In BO > Modules > Check that the module dashactivity is enabled 
+
+Check the existing of the specific UI block in the main Prestashop dashboard page are exist.
```
Comment (Hlavtox) [inline README.md:18]:
```suggestion
Check if specific UI blocks on the main Prestashop dashboard page exist.
```

File: README.md [hunk 4]
```diff
@@ -8,6 +8,19 @@ Check in one glance what is happening on your store with a list of KPI on your d
...
+
+In dashboard page check the existing of Activity overview block
```
Comment (Hlavtox) [inline README.md:20]:
```suggestion
On dashboard page, check if Activity overview block exists.
```

File: README.md [hunk 5]
```diff
@@ -8,6 +8,19 @@ Check in one glance what is happening on your store with a list of KPI on your d
...
+
+Check the different blocks : Online visitors/ Active Shopping Carts /Currently pending /Notifications/Customers & Newsletters /Traffic
```
Comment (Hlavtox) [inline README.md:22]:
```suggestion
Check the different blocks: Online visitors / Active Shopping Carts / Currently pending / Notifications / Customers & Newsletters / Traffic
```

**Review by tleon — DISMISSED** [review]
I agree with the returns. LGTM otherwise.

**Comment by hibatallahAouadni** [comment]
@Hlavtox or @tleon could you please apply the proposed changes 🙏 we're busy 😁 
Thanks in advance 🙏

**Review by Hlavtox — APPROVED** [review]

**Pull request merged**